jpo-ode-svcs/src/main/java/us/dot/its/jpo/ode/importer/parser/PayloadParser.java

```diff
@@ -20,18 +20,31 @@
 import java.io.OutputStream;
 import java.nio.ByteOrder;
 import java.util.Arrays;
+import java.util.HashMap;
+
+import org.apache.tomcat.util.buf.HexUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 import us.dot.its.jpo.ode.util.CodecUtils;
 
 public class PayloadParser extends LogFileParser {
 
-   public static final int PAYLOAD_LENGTH_LENGTH = 2;
+   private static Logger logger = LoggerFactory.getLogger(PayloadParser.class);
+   private static HashMap<String, String> msgStartFlags = new HashMap<String, String>();
 
+   private static final int HEADER_SIZE = 20; 
+   public static final int PAYLOAD_LENGTH = 2;
+   
    protected short payloadLength;
    protected byte[] payload;
+   protected String payloadType;
 
    public PayloadParser() {
       super();
+      msgStartFlags.put("BSM", "0014");
```

Comment:
Constant variables for the flag values?

Reply:
Do we need one for PSM too?

Reply:
Same response as above:
A start flag for PSM messages is not necessary currently, as we don't currently support offloaded PSM messages through log files (it is also not supported by Commsignia/Yunex devices).